### ISSUE TITLE
Default settings navigation order

### ISSUE THREAD

**Issue by mffap** [zitadel/zitadel#8086]
### Preflight Checklist

- [X] I could not find a solution in the existing issues, docs, nor discussions
- [ ] I have joined the [ZITADEL chat](https://zitadel.com/chat)

### Describe your problem

This is rather a suggestion based on doing demos and explaining the settings a lot:

a) Since "Features" are feature toggles and for advanced use cases, they probably can appear under "Other"
b) Some of the "Other" settings feel like they should belong into other categories

### Describe your ideal solution

Proposed grouping of the navigation: 

General Information

- Organization
- Languages
- Audit Trail (3)

Notifications

- SMTP Provider
- SMS/Phone Provider
- Alerts (1)

Login and Access

- Login Behavior and Security
- Identity Providers
- Password complexity
- Lockout
- OIDC Token lifetime and expiration (2)
- Secret Generator (2)

Domain

- Domain settings

Appearance

- Branding
- Message Texts
- Login Interface Texts
- External Links (4) 

Features (5)

- Experimental
- Security

--- 

(1) Rename for clarify, put to the bottom since this is more advanced than SMTP/SMS providers
(2) These feel related to Login and access
(3) "Events" from storage. This is a key feature and should be more prominent. "Audit trail" can be understood without understanding ZITADEL's concept of "events".
(4) External links are related to appearance
(5) Group activating and de-activating features. Experimental are the "feature settings" and security are the "security settings"

### Environment

ZITADEL Cloud

**Comment by mffap** [comment]
@skewis6 I know that you're looking into reworking the console. This was a suggestion from a while ago. Do we still need this issue?

**Comment by skewis6** [comment]
> [@skewis6](https://github.com/skewis6) I know that you're looking into reworking the console. This was a suggestion from a while ago. Do we still need this issue?

@mffap yep the new design makes this redundant. "Default Settings" will be redone, and many of this will have their own pages going foward. Eg Actions, Policies, Storage etc.

The new settings page design is grouped by category also.

This will all be done when we do the upcoming UI overhaul for Console.

I'll close this.
